html/global_attributes.json

```diff
@@ -836,7 +836,10 @@
           }
         }
       },
-      "inputmode": {
+      "
+      
+      
+      ": {
```

Comment:
Whoa, what happened along here?  If we can get these lines reverted back to normal, then I'd say this looks perfect!

Reply:
Ummm - clearly I need a LOT more practice before making my first change! I do appreciate your friendly feedback. I'll close this, stand in the corner, put the dunce hat on, and make an appointment with my counsellor ;-) Cheers